morning/parser.js

```diff
@@ -25,18 +25,14 @@ module.exports = new Parser(function analyseEC(parsedUrl, ec) {
     // http://library.morningstar.com/ArticleSuppot/article?id=1016715
     result.rtype    = 'ARTICLE';
     result.mime     = 'HTML';
-    result.unitid = param.id;
```

Comment:
That not a good idea to remove unitid from parser that detects ARTICLE, is it because that id in query param is not here every the time ? The best would be to assign the unitid whenever it is present.
like this : 
```js
if (result.id) {
 result.unitid = param.id;
}
```

Reply:
Hi,
I have added back param.id
It should be there. Please check
Thanks